core/src/navigation/TopNav.html

```diff
@@ -306,6 +314,11 @@
       },
       burgerClicked() {
         document.body.classList.toggle('lui-leftNavToggle');
+      },
+      userInfoUpdate(userInfo) {
+        if (userInfo) {
```

Comment:
So, in case of some misconfiguration, there will be no errors, just an outdated user info?